serve.js

```diff
@@ -9,13 +9,25 @@
 var http = require("http");
 var resume = require("resume-schema").resumeJson;
 var theme = require("./index.js");
+var path = require("path");
+var fs = require("fs");
 
 var port = 8888;
 http.createServer(function(req, res) {
-    res.writeHead(200, {
-        "Content-Type": "text/html"
-    });
-    res.end(render());
+    var picture = resume.basics.picture;
+    if (picture && req.url.replace(/^\//, "") === picture.replace(/^.\//, "")) {
+        var format = path.extname(picture);
```

Comment:
I'm not familiar with node.js but if you just update your `resume.json` to 

```json
    "picture": "/static/images/my-picture.jpg",
```

don't you think this'd work ? since [here](https://github.com/mudassir0909/jsonresume-theme-elegant/blob/master/jade/profile-card.jade#L27) we just update the `src` attribute with whatever that is being passed by the resume object